src/sentry/replays/post_process.py

```diff
@@ -73,13 +81,27 @@ class ReplayDetailsResponse(TypedDict, total=False):
     info_ids: list[str] | None
     count_warnings: int | None
     count_infos: int | None
+    has_viewed: bool
+
+
+@extend_schema_serializer
+class ReplayViewedByResponse(TypedDict):
+    id: str
+    viewed_by: list[UserSerializerResponse]
 
 
 def process_raw_response(
-    response: list[dict[str, Any]], fields: list[str]
+    response: list[dict[str, Any]],
+    fields: list[str],
+    actor_uid: int | None = None,
```

Comment:
why are we calling this actor_uid inster of just user_id or something?

Reply:
Was worried bout confusion with the user_id field in the replays table. Open to suggestions. I also considered:
- as_user
- request_user_id
- viewed_by_id